x-pack/plugins/threat_intelligence/public/modules/indicators/lib/display_value.ts

```diff
@@ -8,10 +8,11 @@
 import { Indicator, RawIndicatorFieldId } from '../../../../common/types/indicator';
 import { unwrapValue } from './unwrap_value';
 
-export type IndicatorValueExtractor = (indicator: Indicator) => string | null;
-export type IndicatorTypePredicate = (indicatorType: string | null) => boolean;
+type IndicatorDisplayName = [RawIndicatorFieldId, string | null];
```

Comment:
the main change is that the DisplayName extractor now always returns a pair: `[field, value]` to awoid having separate logic for display name field